src/components/FourOhFour/FourOhFour.jsx

```diff
@@ -3,8 +3,12 @@ import styles from './FourOhFour.css';
 
 export default () => (
   <div className={styles.container}>
-    <h1>404</h1>
-    <h2>Page not found.</h2>
-    :(
+    <img src="/assets/images/sad.gif" role="presentation" />
```

Comment:
It's dota 2 image like a items, cosmetics, heroes. I am sure that there are no problems with it.